Enable Conversations live tests

heaths opened the pull request:
The pipeline variable group has already been updated.

heaths committed:
Enable Conversations live tests

heaths commented:
/azp run net - cognitivelanguage - tests

weshaggard commented on sdk/cognitivelanguage/tests.yml:
```diff
@@ -4,6 +4,9 @@ extends:
   parameters:
     EnvVars:
+      CONVERSATIONS_KEY: $(clu-key)
+      CONVERSATIONS_PROJECT: antischOne
```
Is antischOne temporary?

heaths replied:
Temporary for a while, yes. We've been working with several teams across the Language pillar to come up with a unified plan and are pretty close. We'll eventually get these resources provisions with test-resources.json + a test-resources-post.ps1 to create, populate, and train projects.

@AhmedLeithy, one other idea is rather than the postscript, we could also do what Search does like so if that's easier: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/search/Azure.Search.Documents/tests/Utilities/SearchResources.cs

heaths committed:
Fix environment variables

heaths commented:
/azp run net - cognitiveservices - tests

heaths commented:
/azp run net - cognitivelanguage - tests

heaths commented:
Good news: this almost works! @AhmedLeithy the test `AnalyzeConversationWithLanguage` is failing with 400 BadRequest but the message isn't very specific as to why. Could you look into this, please? If we can get that fixed, these changes should re-enable the live tests.

/cc @annatisch @ZulaMostafa @mshaban-msft

heaths committed:
Temporarily disable some tests

Disable a few tests until #26379 is resolved so we can get nightly live tests working again.

heaths commented:
/azp run net - cognitivelanguage - tests

heaths committed:
Fix AnswersFollowupKnowledgeBaseQuestion results

Had a second result come back that was 0.30 confidence, which I've seen in the past as well. Tends to vary whenever the model is retrained. Ask for a higher threshold now.

heaths commented:
/azp run net - cognitivelanguage - tests
